tests/publiccloud/run_ltp.pm

```diff
@@ -141,18 +141,20 @@ sub run {
 
     assert_script_run($log_start_cmd);
 
-    my $cmd = 'perl -I runltp-ng runltp-ng/runltp-ng ';
-    $cmd .= '--logname=ltp_log --verbose ';
-    $cmd .= '--timeout=1200 ';
-    $cmd .= '--run ' . get_required_var('LTP_COMMAND_FILE') . ' ';
-    $cmd .= '--exclude \'' . get_var('LTP_COMMAND_EXCLUDE') . '\' ' if get_var('LTP_COMMAND_EXCLUDE');
-    $cmd .= '--backend=ssh';
+    zypper_call("in -y python3-paramiko python3-scp");
+ 
+    my $cmd = 'python3 runltp-ng/runltp-ng ';
+    $cmd .= "--json-report=$root_dir/result.json ";
+    $cmd .= '--verbose ';
+    $cmd .= '--suite-timeout=2400 ';
```

Comment:
why double the timeout ?

Reply:
Please consider that I executed this below test on openqa (clone of [9562977](https://openqa.suse.de/tests/9562977)) that even **timed out with this 2400 timeout** against the old one 1200: 
https://openqa.suse.de/tests/9588743

I'd even suggest to apply thi change:
```
    my $timeout = get_var('LTP_TIMEOUT', 30 * 60);
```
then:
```
    my $cmd = 'python3 runltp-ng/runltp-ng ';
...
    $cmd .= '--suite-timeout=' . $timeout . ' ';
...

```